test/samplers/test_samplers.py

```diff
@@ -61,20 +94,80 @@ def test_sampler(sampler, num_indices_between_frames):
     )
 
 
-@pytest.mark.parametrize("sampler", (clips_at_random_indices, clips_at_regular_indices))
+@pytest.mark.parametrize("seconds_between_frames", [None, 3])
```

Comment:
It would be nice to write a test that derives seconds_between_clip_starts from the video duration like a real user would. In that test we should also make sure that there are no repeated frames in the sampled frames.

Reply:
I don't think users will derive `seconds_between_clip_starts` from the video duration.